app/javascript/retrospring/features/announcement/index.ts

```diff
@@ -0,0 +1,14 @@
+import registerEvents from 'utilities/registerEvents';
+import closeAnnouncementHandler from './close';
+
+export default (): void => {
+  registerEvents([
+    { type: 'click', target: '.announcement button.close', handler: closeAnnouncementHandler, global: true },
```

Comment:
```suggestion
    { type: 'click', target: document.querySelector('.announcement button.close'), handler: closeAnnouncementHandler },
```

If you are using `turbolinks:load` as an event for a feature, you can't use global events (or, it doesn't work particularly well).

Switching the event to `DOMContentLoaded` or removing `global: true` and query-selecting an element in `target`, like I did in this suggestion.